java/clients/src/main/java/sleeper/clients/QueryWebSocketClient.java

```diff
@@ -57,47 +47,30 @@
 import java.util.Optional;
 import java.util.Set;
 import java.util.TreeMap;
-import java.util.UUID;
 import java.util.function.Consumer;
-import java.util.function.Supplier;
 import java.util.stream.Collectors;
 
 import static sleeper.configuration.properties.instance.CdkDefinedInstanceProperty.QUERY_WEBSOCKET_API_URL;
 
-public class QueryWebSocketClient extends QueryCommandLineClient {
+public class QueryWebSocketClient {
     private final String apiUrl;
     private final Client client;
+    private final ConsoleOutput out;
```

Comment:
This is still using the console for output. Don't we need to get Record objects?

Reply:
Did you want to make a separate issue to move the ConsoleOutput object to QueryWebSocketCommandLineClient?

Reply:
I made the class return record objects in this PR https://github.com/gchq/sleeper/pull/2180 i can update that PR to move the console output

Reply:
I don't see how those things fit together. What matters is the interface that will be used by the system test code to make a query and get the results. It looks like that's what this issue is for? I don't see how this or the linked issues would separate that from the command line.

Reply:
It's making more sense where you're going with this now with the other draft PR. Thanks.